modelzoo/DLRM/train.py

```diff
@@ -334,262 +658,148 @@ def get_arg_parser():
     parser.add_argument('--input_layer_partitioner',
                         help='slice size of input layer partitioner. units MB',
                         type=int,
-                        default=0)
+                        default=8)
     parser.add_argument('--dense_layer_partitioner',
                         help='slice size of dense layer partitioner. units KB',
                         type=int,
+                        default=16)
+    parser.add_argument('--tf', \
+                        help='Use TF 1.15.5 API and disable DeepRec feature to run a baseline.',
+                        action='store_true')
+    parser.add_argument('--smartstaged', \
+                        help='Whether to enable smart staged feature of DeepRec, Default to True.',
+                        type=boolean_string,
+                        default=False) #TODO: Defautl to True
```

Comment:
enable and set to true by default

Reply:
@chenzhiyin  Please help to update the code according to Duyi's comments. Thanks.